magma/lib/magma/storage.rb

```diff
@@ -19,9 +19,6 @@ def download_url project_name, path
     def upload_url(project_name, path, params={})
     end
 
-    def setup_uploader(model, name, type)
```

Comment:
This, and the uploader classes, are no longer referenced or used by any active code paths.